packages/docs/src/routes/docs/(qwikcity)/caching/index.mdx

```diff
@@ -47,10 +47,6 @@ export default component$(() => {
 
 With the above setup, you will not only have better performance (pages are always served instantly from cache), but you can also have significantly decreased hosting costs, as our server or edge functions only need to run at most once every 5 seconds per page.
 
-## `cacheControl`
-
-Any method that takes a [request event](https://qwik.dev/docs/advanced/request-handling/#request-event) can call `request.cacheControl` to set the cache control headers for the response:
-
 ```tsx title="src/routes/layout.tsx"
```

Comment:
I would keep the h2 and paragraph here. Otherwise the code snippet looks like it's coming out of nowhere.